plugins/confluence/commands/sync-meeting-notes.md

```
@@ -0,0 +1,143 @@
+---
+description: Publish meeting notes or agendas to Confluence with structured formatting
+argument-hint: "<space-key> <title> [--parent-id <id>] [--update <page-id>]"
+---
+
+## Name
+confluence:sync-meeting-notes
+
+## Synopsis
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)